### PR TITLE
feat: add info about used identifier

### PR DESCRIPTION
**Pull request by mszekiel**
## Related Issue or Design Document

Implements [#3288](https://github.com/ory-corp/cloud/issues/3288)

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/stories/Ory/Auth.stories.tsx [hunk 1]
```diff
@@ -87,6 +88,20 @@ LoginAuthCardPasswordless.args = {
   },
 }
 
+export const LoginAuthCardRefresh = Template.bind({})
+
+LoginAuthCardRefresh.args = {
+  title: "Two-Factor Authentication",
```
Comment (Benehiko) [inline src/stories/Ory/Auth.stories.tsx:94]: Wouldn't the title here be "Confirm it is you"

File: src/react-components/ory/user-auth-card.tsx [hunk 2]
```diff
@@ -290,6 +294,7 @@ export const UserAuthCard = ({
               data-testid={`${flowType}-flow`}
             >
               {$flow}
+              {showIdentifier && <IdentifierInfo flow={flow} />}
```
Comment (Benehiko) [inline src/react-components/ory/user-auth-card.tsx:297]: Not sure if this belongs here, what happens if i only have a webauthn as a password?

Also this isn't part of the 2FA flow - even though it contains similar data.

Comment (mszekiel) [reply src/react-components/ory/user-auth-card.tsx:297]: As we have `!isTwoFactor()` it's not triggering for 2FA flow, like it's reflected on Chromatic tests.

File: src/stories/Ory/Auth.stories.tsx (hunk 1 above)
Comment (mszekiel) [reply src/stories/Ory/Auth.stories.tsx:94]: To be changed also in kratos-selfservice.

File: src/react-components/ory/user-auth-card.tsx (hunk 2 above)
Comment (Benehiko) [reply src/react-components/ory/user-auth-card.tsx:297]: ah okay nice!

**Review by Benehiko — COMMENTED** [review]
It would be nice to have a test coverage for this in https://github.com/ory/elements/blob/main/src/react-components/ory/user-auth-card.spec.tsx

File: src/react-components/ory/user-auth-card.tsx [hunk 3]
```diff
@@ -113,6 +115,8 @@ export const UserAuthCard = ({
       hasWebauthn(flow.ui.nodes) ||
       hasLookupSecret(flow.ui.nodes))
 
+  const showIdentifier = hasHiddenIdentifier(flow.ui.nodes)
```
Comment (jonas-jonas) [inline src/react-components/ory/user-auth-card.tsx:118]: Would be good to have a comment explaining why `hasHiddenIndentifier` leads to showing the identifier. I was confused for a second.

**Review by jonas-jonas — APPROVED** [review]
LGTM, just one comment request :)

**Review by Benehiko — APPROVED** [review]
Nice :heart:

**Pull request merged**